src/main/java/edu/usf/cutr/grha/gtfs/GtfsWriterMain.java

```diff
@@ -0,0 +1,85 @@
+package edu.usf.cutr.grha.gtfs;
+
+import edu.usf.cutr.grha.model.ChicagoTncData;
+import org.onebusaway.gtfs.model.Agency;
+import org.onebusaway.gtfs.model.AgencyAndId;
+import org.onebusaway.gtfs.model.Route;
+import org.onebusaway.gtfs.model.Stop;
+import org.onebusaway.gtfs.serialization.GtfsWriter;
+
+import java.io.File;
+import java.util.List;
+
+public class GtfsWriterMain {
```

Comment:
I wouldn't call this `Main` - usually that's reserved for files with a `void main()` entry point.  How about just `GtfsWriter`? Or something else short.